cli/command/container/opts.go

```diff
@@ -679,6 +679,14 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 		return nil, err
 	}
 
+	if hostConfig.NetworkMode.IsBridge() || hostConfig.NetworkMode.IsUserDefined() {
```

Comment:
Before I forget to comment; we need to check if this does what we expect it to.


I recall I wanted to use some of these functions on the client-side, but the code was written "platform specific", so `IsUserDefined()` on a Linux client potentially wouldn't detect a "user-defined" network if the daemon is Windows (or vice-versa).